spec/features/proposal/collection_draft_proposals/collection_draft_proposal_submit_spec.rb

```diff
@@ -97,11 +94,40 @@
 
       it 'provides the correct error message' do
         within '#proposal-status-display' do
-          expect(page).to have_content('PROPOSAL STATUS:')
-          expect(page).to have_content('DONE')
+          expect(page).to have_content('Done')
         end
         expect(page).to have_content 'Collection Draft Proposal was not rescinded successfully'
       end
     end
   end
+
+  context 'when looking at a delete metadata request' do
+    before do
+      set_as_proposal_mode_mmt(with_required_acl: true)
+      @collection_draft_proposal = create(:full_collection_draft_proposal, draft_request_type: 'delete')
+      mock_submit(@collection_draft_proposal)
+      visit collection_draft_proposal_path(@collection_draft_proposal)
+    end
+
+    it 'can be rescinded and deleted' do
+      short_name = @collection_draft_proposal['short_name']
```

Comment:
discussed having these actions in a `before` block, to keep a consistent structure for our tests.